### DISCUSSION TITLE
What is "Lookup" in chat?

### DISCUSSION THREAD

**Discussion by ariel-gil** [brianpetro/obsidian-smart-connections#924] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/761713f9-7187-4ad5-80b6-a5c017c64d00)
What is this "lookup"? it does seem to change depending on the prompt. Fyi, I am running it on Ollama (Llama3.2 3B). The response afterwards does seem to make sense, the context/embedding does seem to be processed correctly. 
For ex, with a different folder requested, it gave this - 
![image](https://github.com/user-attachments/assets/2ae02c4a-8ff5-4235-8214-871d465b6b6f)

**Comment by brianpetro** [comment, marked as answer]
It can be thought of as like a search query made by the AI 🌴
